worlds/kh1/Items.py

```diff
@@ -0,0 +1,534 @@
+from typing import Dict, NamedTuple, Optional, Set
+
+from BaseClasses import Item, ItemClassification
+
+
+class KH1Item(Item):
+    game: str = "Kingdom Hearts"
+
+
+class KH1ItemData(NamedTuple):
+    category: str
+    sub: str = "None"
+    code: Optional[int] = None
+    classification: ItemClassification = ItemClassification.filler
+    max_quantity: int = 1
+    weight: int = 1
+
+
+def get_items_by_category(category: str, disclude: list) -> Dict[str, KH1ItemData]:
+    item_dict: Dict[str, KH1ItemData] = {}
+    for name, data in item_table.items():
+        if data.category == category and all(x not in name for x in disclude):
+            item_dict.setdefault(name, data)
+
+    return item_dict
+
+
+item_table: Dict[str, KH1ItemData] = {
+    "Victory":                           KH1ItemData("Victory",          code = 264_0000, classification = ItemClassification.progression,  max_quantity = 1, weight = 10),
```

Comment:
Is this meant to be an actual item, or this meant to be the victory event?

Reply:
It's an item, having the item causing a Victory send.  Generation can place the item in several places depending on your goal condition